.github/workflows/specs.yml

```
@@ -0,0 +1,81 @@
+name: Test extractions for `hacpsec/specs`
+
+on:
+  push:
+    branches:
+    - '*'
```

Let's run this on PRs and on demand as well.
At some point this shouldn't run on branches anymore, but that's ok for now.

```suggestion
  pull_request:
  workflow_dispatch:
```